packages/query-parser/package.json

```
@@ -69,5 +69,8 @@
     "prettier": "^2.3.2",
     "sinon": "^9.2.3",
     "typescript": "^5.0.4"
+  },
+  "peerDependencies": {
+    "bson": "^6.8.0"
```

Thanks for opening the pr!
Should we match the `peerDependencies` of `shell-bson-parser` here? https://github.com/mongodb-js/devtools-shared/blob/ca8aea2deb73026172732c1cf5cd8d36208ec440/packages/shell-bson-parser/package.json#L51
```suggestion
    "bson": "^4.6.3 || ^5 || ^6"
```

---

Thanks, I did the fix